mlrun/api/utils/singletons/k8s.py

```diff
@@ -1,17 +1,5 @@
-from mlrun.k8s_utils import K8sHelper
-
-# TODO: something nicer
-k8s: K8sHelper = None
+from mlrun.k8s_utils import K8sHelper, get_k8s_helper
 
 
 def get_k8s() -> K8sHelper:
-    global k8s
-    return k8s
-
-
-def initialize_k8s():
-    global k8s
-    try:
-        k8s = K8sHelper()
-    except Exception:
-        pass
+    return get_k8s_helper(silent=True)
```

Comment:
Here we are under the `api` module, and this is the module for `singletons` for the API.
the `k8s_utils.py` which `get_k8s_helper()` comes from, and as you said already manages a singleton, is used both by the API and the SDK and therefore can't be here.
When I wrote this file `k8s.py` I didn't notice that there are actually double management of singletons, only now I did, so fixed it in this PR, and made this thing here just a proxy to there.
I can basically remove the proxy and reference the original from everywhere, but It will make tons of changes in a lot of files so I prefer not to, also I think that maybe eventually there will be something here designeted only for the API usage so another reason to currently keep it